compiler-core/src/type_/snapshots/gleam_core__type___tests__empty_func_warning_test.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: compiler-core/src/type_/tests.rs
+expression: "pub fn main() { foo() }\n        pub fn foo() { }"
```

Comment:
```gleam
pub fn main() { }
```

... should probably be sufficient for the testcase + snapshot and should return:

```
pub fn main() {
  todo
}
```
... I guess.

Reply:
Thanks, I will make a change.

I'm not sure how to make it return `pub fn main() {
  todo
}` since the `assert_warning` macro returns the source code and we are modifying the AST. Would you mind elaboring on that?